packager.sh

```diff
@@ -1214,7 +1214,7 @@ GEM_BUILD_SRC="${GEM_BUILD_ROOT}/${GEM_DEB_PACKAGE}"
 mksafedir "$GEM_BUILD_ROOT"
 mksafedir "$GEM_BUILD_SRC"
 
-git archive HEAD | (cd "$GEM_BUILD_SRC" ; tar xv ; rm -rf rpm)
+git archive HEAD | (cd "$GEM_BUILD_SRC" ; tar xv ; rm -rf rpm ; rm -rf $(find . -type d -name tests))
```

Comment:
OT: I see you are removing `rpm`. We recently added `docker` too: does this mean that you have also to remove it? Why we don't simply take files coming out by the python package that are already filtered?

Reply:
Currently we are packaging directly sources using git archive as source instead using python package, we can address this problem in a separated PR